features/CLI/CommandLine.java

```diff
@@ -0,0 +1,186 @@
+import java.util.Scanner;
+
+
+public class CommandLine {
+
+	public static void main(String[] args) {
+		Scanner cmdLine = new Scanner(System.in);
+		System.out.println("Hello, and welcome to Team mike's UML editor.");
+		System.out.println("To exit the program, type quit");
+		System.out.println("To see all the commands available, type help");
+
+
+		while(true)
+		{
+			String line = cmdLine.nextLine();
+			//parse command line string into a list of commands by spaces
+			String[] commands = line.split(" ");
+			//get first command
+			String command = commands[0];
+
+			if (command.equals("quit"))
+			{
+				break;
+			}
+			
+			switch(command)
```

Comment:
This creates a copy of command, compares it to quit, and then used in the switch statement's argument. Might be a bit nit-picky but would it be worth changing it to something like:

```
if (commands[0].equals("quit"))
{
	break;
}
switch(commands[0])...
```

Reply:
It basically boils down to whether you'd like to see a variable or an index to an array. 

I'll change it in the next commit just so we can see how it looks and how we feel about it.